.github/workflows/publish_pypi.yml

```diff
@@ -24,9 +27,6 @@ jobs:
         with:
           python-version: 3.8
 
-      - name: Install cibuildwheel
-        run: python -m pip install cibuildwheel==2.12.3
-
```

Comment:
`pipx` can get us updated versions of `cibuildwheel`. Otherwise, we could use its action, which is compatible with Dependabot, which has been enabled already